build/source/engine/ffile_info.f90

```diff
@@ -153,15 +147,19 @@ subroutine ffile_info(nGRU,err,message)
   ! set nVar attribute
   forcFileInfo(iFile)%nVars = nVar
 
-  ! allocate space
-  allocate(forcFileInfo(iFile)%var_ix(nVar), stat=err)
-  if(err/=0)then
-   message=trim(message)//'problem allocating space for structure element'
-   err=20; return
-  endif
+  ! ensure allocatable structure components are deallocated
```

Comment:
Is there a reason to include all of this in the "read faster" pull request?

Reply:
Agreed - can we split this so the chunksize PR is separate and its own thing? Much easier to track if we need to modify it later (or roll it back).

Reply:
It should be easy to split the two commits into separate PRs.